tools/download-content-markdown.mjs

```diff
@@ -0,0 +1,47 @@
+/*  fetches existing Sharepoint documents as Markdown and stores them in a folder structure.
+    I use this to research existing blocks for certain content.
+ */
+
+import fetch from 'node-fetch';
+import fs from "fs";
+import path from "path";
+
+async function createFolder(url) {
+    const folderPath = `./mirror${new URL(url).pathname.replace('/.*$', '')}`;
+    try {
+        await fs.promises.mkdir(folderPath, {recursive: true});
+    } catch (ignore) {
+    }
+    return folderPath;
+}
+
+function getLocale(url) {
+  if (url.includes('www.zeiss.de')) {
+    return 'de';
+  }
+  return 'en';
+};
+
+async function getPaths() {
+    const response = await fetch("https://main--zeiss--hlxsites.hlx.page/drafts/import-report.json");
```

Comment:
I think it would be more robust to fetch from query-index.json which would get auto-updated on new page additions.